lib/event_sourcery/memory/projector.rb

```diff
@@ -0,0 +1,25 @@
+module EventSourcery
+  module Memory
+    module Projector
+
+      def self.included(base)
+        base.include(EventSourcery::EventProcessing::EventStreamProcessor)
+        base.include(InstanceMethods)
+        base.class_eval do
+          alias_method :project, :process
+          class << self
+            alias_method :project, :process
+            alias_method :projects_events, :processes_events
+            alias_method :projector_name, :processor_name
+          end
+        end
```

Comment:
:+1: We should extract event sink boilerplate and helpers such as projector and reactor back into ES and just have the storage layer and its config in this gem.